docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md

```diff
@@ -0,0 +1,81 @@
+---
+id: version-1.8.0-deploy_agw_using_juju
+title: Deploy AGW using Juju
+hide_title: true
+original_id: deploy_agw_using_juju
+---
+
+# Deploy Charmed Magma Access Gateway
+
+## Requirements
+
+### Management machine
+
+- Ubuntu machine with internet access
+
+### Access Gateway machine
+
+The Access Gateway must be installed on an Ubuntu machine with the following specifications:
```

Comment:
```suggestion
The Access Gateway must be installed on an machine with the following specifications:
```

Reply:
In this case, Ubuntu is actually required. We can't run AGW on other OSes.

Reply:
The OS is listed in the bullet list below

Reply:
True. Removed.